tests/test_models.py

```diff
@@ -300,9 +307,33 @@ def test_search_criterion_dto_mapping(self):
             is criterion.contains_records_in_integration_schemas
         )
         assert dto.contains_records_in_tables is criterion.contains_records_in_tables
-        assert dto.contains_records is criterion.contains_records
+        assert dto.contains_records == Unset
```

Comment:
I think Unset is a singleton?

```suggestion
        assert dto.contains_records is Unset
```